site/source/docs/api_reference/html5.h.rst

```diff
@@ -2405,3 +2405,9 @@ Functions
 .. c:function:: void emscripten_throw_string(const char *utf8String)
 
   Invokes JavaScript throw statement and throws a string.
+
+.. c:function:: void emscripten_unwind_stack(void)
```

Comment:
Something doesn't feel right to me about this name. Perhaps `emscripten_halt_and_unwind`? Or `emscripten_halt_current_execution`? Those aren't great either...

Reply:
Saying "halts execution" sounds to me very much the same as "aborts" or "terminates"? But the idea is not to terminate the page. `emscripten_unwind_and_yield()`? `emscripten_unwind_to_event_loop()`? `emscripten_unwind_to_browser_event_loop()`?

Reply:
Good point, yeah. I like the last names, how about `emscripten_unwind_to_js_event_loop` (as it may be outside of the browser, and in node.js)?

Reply:
Sounds good!